docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,1440 @@
+   },
+   "source": [
+    "## Concepts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "The FHIR spec is large and daunting, so we cannot cover all of the major concepts in FHIR. However, this section and the next should give you all you need to understand what you're doing in the tutorial. \n",
+    "\n",
+    "📘A good resource to learn [FHIR Basics](https://smilecdr.com/docs/tutorial_and_tour/fhir_basics.html#fhir-basics)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "### FHIR Resource "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "A term that describes a basic entity in the FHIR specification. A resource definition can be in JSON or XML format, but for the purposes of this guide, we will always use JSON.\n",
+    "\n",
+    "📘Learn more about [FHIR Resources](https://www.hl7.org/fhir/resource.html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "**Examples of resources are:**\n",
+    "\n",
+    "`Patient` - which represents a person in a healthcare system\n",
+    "\n",
+    "`Specimen` - which represents a biosample drawn from a patient\n",
+    "\n",
+    "`StructureDefinition` - this is different from the first two. It describes the model or definition for various\n",
+    "healthcare entities such as `Patient`, `Specimen`, etc.\n",
+    "\n",
+    "**A `Patient` resource might look like this:**\n",
+    "```json\n",
+    "{\n",
+    "    \"resourceType\":\"Patient\",\n",
+    "    \"id\": \"PT-001\",\n",
+    "    \"meta\": {\n",
+    "        \"profile\": [\"http://fhir.kids-first.io/StructureDefinition/Patient\"]\n",
+    "    },\n",
+    "    \"gender\": \"male\",\n",
+    "    \"name\": [\n",
+    "        {\n",
+    "            \"use\": \"usual\",\n",
+    "            \"family\":\"Smith\",\n",
+    "            \"given\": \"Jack\",\n",
+    "            \"period\": {\n",
+    "                \"end\": \"2001-01-01\"\n",
+    "            }\n",
+    "        }\n",
+    "    ]\n",
+    "}\n",
+    "``` "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "### Resource Categories"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "There are different categories of FHIR resources such as \n",
+    "\n",
+    "- `Conformance Resource` \n",
+    "- `Terminology Resource`\n",
+    "- `Individual Resource`\n",
+    "- `Diagnosic Resource`\n",
+    "- ...many more\n",
+    "\n",
+    "However, the most important categories of resources for the purposes of this tutorial are `Conformance Resources` and `Terminology Resources` because these are the ones we will create in order to build a FHIR data model. "
```

Comment:
After reading until here, I've got a concept conflict about the definition of a "resource." So far as I understand, it seems we call a data model a resource in FHIR (for example, Patient, Specimen, Observation, etc.) and do these resources fall into one of the categories of resources (for example, Observation is a diagnostic resource)? Or, is a resource described by these categories of resources? Say, given a patient resource, some parts of the resources are conformance resources whereas the other parts are terminology resources?

Reply:
Thanks @liberaliscomputing, good points. The FHIR spec itself is very confusing. All entities in FHIR are Resources. Then there are categories of Resources which are just groupings of different Resource types. I can make a diagram to clarify this 👍